packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -735,6 +740,11 @@ class SemanticsOwner extends ChangeNotifier {
     assert(action != null);
     final SemanticsActionHandler handler = _getSemanticsActionHandlerForId(id, action);
     handler?.performAction(action);
+
+    // Default actions if no [handler] was provided.
+    if (handler == null && action == SemanticsAction.showOnScreen && _nodes[id]._showOnScreen != null) {
```

Comment:
We're testing handler against null twice here. It would be more readable (and would make the comment unnecessary) if you just did this as an if/else.

But it's not clear to me why we don't call showOnScreen if there's a handler. A comment explaining _that_ would be useful. :-)